src/index.css

```diff
@@ -200,12 +189,20 @@ input[type=number]::-webkit-outer-spin-button {
   min-width: 100px;
 }
 
+/* FEEDBACK BUTTON */
+#feedback_fixed {
+    position: fixed;
+    top: 147px;
+    left: -57px;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+}
+
 /* RESET BUTTON */
 #reset_fixed {
     position: fixed;
-    top: 253px;
+    top: 270px;
     left: -21px;
-    z-index: 3000;
```

Comment:
@migueldcepeda Just to point out, I got rid of the z-index property on both of the side buttons because it seemed confusing to be able to select one of them while a modal was already open (and all the modals now allow submitting feedback as an option). Does that sound reasonable and was there any other reason for setting the z-index?

Reply:
@ethanbb That makes total sense and something I noticed but didn't want to mess with. I believe the z-index was something that was coded in prior to the edits I made.

Reply:
Yes, sorry, the z-index was there when the button was horizontal to make sure that the buttons were never covered by the questions.